module1/tasks.js

```diff
@@ -0,0 +1,189 @@
+// ==== Module#1 Basics ====
+mocha.setup('bdd');
+const expect = chai.expect;
+
+// ==== Change the capitalization of all letters in a string ====
+const changeCase = (str) => { 
+  let result = '';
+  for (let i = 0; i < str.length; i++) {
+    const char = str.charAt(i);
+    if(isUpperCase(char)) {
+      result += char.toLowerCase();
+    } else {
+      result += char.toUpperCase();
+    }
+  }
+  return result;
+};
+
+function isUpperCase(char) {
+  return char == char.toUpperCase();
+}
+function convertChar(c) {
+  return isUpperCase(c) ? c.toLowerCase() : c.toUpperCase();
+}
+
+const changeCase2 = (str) => {
+  return [...str]  //transform string to array of chars
+    .map(c => convertChar(c))
+    .join("");  //collect
+};
+
+
+describe('changeCase', () => {
+  it('should change the capitalization of all letters in a given string', () => {
+    expect(changeCase('21century')).to.equal('21CENTURY');
+    expect(changeCase('Hybris')).to.equal('hYBRIS');
+    expect(changeCase2('21century')).to.equal('21CENTURY');
+    expect(changeCase2('Hybris')).to.equal('hYBRIS');
+  });  
+});
+
+// ==== Filter out the non-unique values in an array ====
```

Comment:
It works but it's not a good solution. Please, read about Set is JS and think how it can be applied here.